docs/docs/dev_docs/getting_started/noir.md

```diff
@@ -16,6 +16,8 @@ Noir is designed for accessible and flexible development of provable programs. I
 Noir is a public good for developing ZK-provable programs and is independent from building products on the existing Aztec network.
 :::
 
+While "Vanilla" Noir will be used to write circuits, we have created a library on top of Noir that will be used to write smart contracts on top of the Aztec Sandbox. Since noir files use the `.nr` extension, we are calling this library "Aztec.nr".
```

Comment:
Is this page a duplicate of the landing page of the proper Noir docs? If so, we should probably replace all the text of this page with a link to Noir's docs.